css-squid/src/test/java/org/sonar/css/CssAstScannerTest.java

```diff
@@ -78,4 +78,16 @@ public void atRules() {
     assertThat(file.getInt(CssMetric.AT_RULES)).isEqualTo(1);
   }
 
+  @Test
+  public void functions() {
+    SourceFile file = CssAstScanner.scanSingleFile(new File("src/test/resources/metrics/metric.css"));
+    assertThat(file.getInt(CssMetric.FUNCTIONS)).isEqualTo(6);
+  }
+
+  @Test
+  public void complexity() {
+    SourceFile file = CssAstScanner.scanSingleFile(new File("src/test/resources/metrics/complexity.css"));
+    assertThat(file.getInt(CssMetric.COMPLEXITY)).isEqualTo(21);
+  }
+
```

Comment:
I don't really know if it's worth it but here it is: https://github.com/racodond/sonar-css/commit/d7b9fde20474587bcea599d266eaa33d905f988d